jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XMLStreamWriterFilter.java

```diff
@@ -53,6 +53,26 @@ public void writeStartDocument() throws XMLStreamException {
         writer.writeStartDocument();
     }
 
+    private boolean isWhiteSpace(char c){
+        // all char points between 0 and 31 excluding the allowed white spaces (9=TAB, 10=LF, 13=CR)
+        if (c >= 0 && c<=31 && c != 9 && c != 10 && c != 13)
```

Comment:
instead of comment _9=TAB, 10=LF, 13=CR_ - why not use `\t`, `\n` and `\r`?
<sup>https://docs.oracle.com/javase/specs/jls/se12/html/jls-3.html#jls-3.10.6</sup>